master/internal/task/allocation_intg_test.go

```diff
@@ -192,7 +192,9 @@ func setup(t *testing.T) (
 		TaskID:       task.TaskID,
 		AllocationID: model.AllocationID(fmt.Sprintf("%s.0", task.TaskID)),
 		SlotsNeeded:  2,
-		Preemptible:  true,
+		Preemption: sproto.PreemptionConfig{
+			Preemptible: true,
+		},
```

Comment:
Same as above, we're not setting timeout duration here so it will default to 0.

Reply:
test isn't impacted by preemption timeout